test/test-najax.js

```diff
@@ -343,6 +364,22 @@ function createSuccess (done) {
   }
 }
 
+function jsonSuccess (done) {
+  return function (data, statusText) {
+    expect(data.test).to.equal('ok')
```

Comment:
I believe that `data.test` is right. `jsonSuccess` is used in the new test on line 98, nock replies with `{'test': 'ok'}`, the whole object becomes `data` and we are looking at the `test` prop.

Of course, without the `nock.cleanAll()` fix or my extra najax call fix, then this test receives the wrong nock and `expect(data.test)` fails.